gltfTutorial/README.md

```diff
@@ -0,0 +1,18 @@
+#glTF Tutorial

```

Comment:
Perhaps add an **Acknowledgements** section to the bottom of this .md file and add everyone who contributes content, provides feedback, edits, etc.  This may help encourage contributions and tie together the glTF community.